docs/visual-basic/getting-started/whats-new.md

```diff
@@ -38,48 +37,104 @@ translation.priority.ht:
   - "zh-tw"
 ---
 # What's new for Visual Basic
-This page lists key feature names for each version of Visual Basic with descriptions of the new and enhanced features in the lastest version of the language.  
-  
-## Previous versions  
- Visual Basic / Visual Studio .NET 2002  
- First release  
-  
- Visual Basic / Visual Studio .NET 2003  
- Bit shift operators, loop variable declaration  
-  
- Visual Basic / Visual Studio .NET 2005  
- `My` type and helper types (access to app, computer, files system, network  
-  
- Visual Basic / Visual Studio .NET 2008  
- Language Integrated Query (LINQ), XML literals, local type inference, object initializers, anonymous types, extension methods, local `var` type inference, lambda expressions, `if` operator, partial methods, nullable value types  
-  
- Visual Basic, Visual Studio .NET 2010  
- Auto-implemented properties, collection initializers, implicit line continuation, dynamic, generic co/contra variance, global namespace access  
-  
- Visual Basic / Visual Studio .NET 2012  
- `Async` / `await`, iterators, caller info attributes  
-  
- Visual Basic / Visual Studio .NET 2013  
- technology previews of .NET Compiler Platform (“Roslyn”)  
-  
- Visual Basic / Visual Studio .NET 2015  
- Current version, see below  
+
+This topic lists key feature names for each version of Visual Basic, with detailed descriptions of the new and enhanced features in the lastest version of the language.
   
-## Current version  
- [Nameof](../../csharp/language-reference/keywords/nameof.md)  
+## Current Version
+
+Visual Basic / Visual Studio .NET 2017   
+For new features, see [Visual Basic 2017](visual-basic-2017)
```

Comment:
I think the link targets should start with a '#'. It doesn't appear to work right now.